frontend/src/v5/store/revisions/revisions.redux.ts

```diff
@@ -60,7 +63,23 @@ export const fetchSuccess = (state = INITIAL_STATE, {
 	...state,
 	revisionsByContainer: {
 		...state.revisionsByContainer,
-		[containerId]: revisions,
+		[containerId]: revisions.map((revision) => prepareRevisionData(revision)),
```

Comment:
Instead of doing this here, do it in the selectors . That way no matter which source changed/added the revision, the data is always formatted correctly for the rest of the app,